simulator.py

```diff
@@ -370,11 +451,12 @@ def symptoms(self):
     @property
     def infectiousness(self):
         if self.is_sick: 
-            time_since_sick = self.infection_timestamp # TODO: env passing! should be: env.timestamp - self.infection_timestamp 
+            time_since_sick = self.env.timestamp - self.infection_timestamp 
```

Comment:
I think we should do `(self.env.timestamp - self.infection_timestamp).days`
it will simplify the comparison and this line `days_sick = int(str(time_since_sick)[0])` which is confusing